src/dashboard/src/installer/steps.py

```diff
@@ -64,10 +58,12 @@ def setup_pipeline(org_name, org_identifier, site_url):
     helpers.set_setting("dashboard_uuid", dashboard_uuid)
 
     # Update Archivematica version in DB
-    set_agent_code("Archivematica-" + get_version(), pk=1)
+    Agent.objects.default_system_agent().update(
+        identifiervalue=get_preservation_system_identifier()
+    )
 
     if org_name != "" or org_identifier != "":
-        agent = get_agent()
+        agent = Agent.objects.default_organization_agent()
```

Comment:
Could we also use `default_organization_agent` here? https://github.com/artefactual/archivematica/blob/c37d5af3edcf757670c4bd03f3c9a29b8e2cc73c/src/dashboard/src/components/administration/views.py#L479